test/unit/org/apache/cassandra/tools/ToolRunner.java

```diff
@@ -196,52 +186,6 @@ public int waitFor() throws InterruptedException
         return this;
     }
 
-    private void watchIO()
-    {
-        OutputStream in = process.getOutputStream();
-        InputStream err = process.getErrorStream();
-        InputStream out = process.getInputStream();
-        while (true)
-        {
-            boolean errHandled;
-            boolean outHandled;
-            try
-            {
-                if (stdin != null)
-                {
-                    IOUtils.copy(stdin, in);
```

Comment:
Also, this assumes the stream will one day close... to enforce that I am adding logic to try to close stdin when the process exists (else the thread blocks forever)